tests/core/test_multi_cluster_app.py

```diff
@@ -315,6 +359,27 @@ def wait_for_roles_to_be_updated(admin_mc, mcapp, roles, timeout=60):
         interval *= 2
 
 
+def wait_for_projects_to_be_updated(admin_mc, mcapp, projects, timeout=60):
```

Comment:
This duplicates a lot of logic that already exists, use `wait_for` in conftest.py and pass in the check function along with an error message that includes both the desired and current state of the projects on the app. This will help future debugging of test failures.